frontend/src/pages/Community.spec.js

```diff
@@ -72,6 +72,7 @@ describe('Community', () => {
                 messagesCount: 0,
                 deniedAt: null,
                 deniedBy: null,
+                moderatorId: 69,
```

Comment:
Does this moderatorId make sence?
In my opinion we can have a contributionListItem without moderatorId, in case of state === 'IN_PROGRESS' or 'PENDING'.

Reply:
Of course it can be null